walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +313,167 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void onlyMandatoryConsideredIfOptionalAreBusy() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times.
+    //
```

Comment:
I would add `Optional: |--------C-------|`, to make it clear what the scenario is in this test.
Same comment for the other tests you added.